simtools/simtel/simtel_histograms.py

```diff
@@ -148,8 +148,8 @@ def _plotCombinedHistograms(self, figName):
         pdfPages = PdfPages(figName)
         for iHist, histo in enumerate(self.combinedHists):
 
-            # Test case: processing only 1/5 of the histograms
-            if self._isTest and iHist % 5 != 0:
+            # Test case: processing only 1/10 of the histograms
```

Comment:
Interesting. Why not 7 or 3?

Reply:
Based on how long the test took to run on my laptop. I was tired of waiting so I reduced it. The test doesn't actually look at the output so we could have simply printed just one histogram, but this is just as good.